tools/CMakeLists.txt

```diff
@@ -1,7 +1,39 @@
-add_executable(
-	HogMaker
-	HogMaker/HogFormat.cpp
-	HogMaker/HogMaker.cpp
+#cmake_minimum_required(VERSION 3.28)
+
+set(HEADERS
+  HogEdit/string_hash.h
+  HogEdit/HogFormat.h
+  HogEdit/IOOps.h)
+
+set(CPPS
+  HogEdit/HogFormat.cpp
+  HogEdit/HogEdit.cpp)
+
+project(HogEdit
+  LANGUAGES C CXX
+  VERSION 0.1.0)
```

Comment:
Same here. We using HogMaker/HogEdit as integral part of Descent3 project (it's part of our building process) and not intending release tools as separate project.

Reply:
Well that seems shortsighted.

Reply:
Anyway, if this is how you are going to treat it then I won't mess with it. It's all yours, now and forever.